Directory.Build.props

```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);NU5125;CS0618</NoWarn>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <LangVersion>10.0</LangVersion>
+    <NetFrameworkMinimum>net472</NetFrameworkMinimum>
```

Comment:
It's better to not overwrite a property that is defined by the Arcade Sdk for consistency: https://github.com/dotnet/arcade/blob/00d6decc59f5030c2399a64fd3e4f6e8e11bacca/src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props#L29